src/include/mpir_coll.h

```diff
@@ -456,11 +456,11 @@ int MPIR_Iallgather_sched_intra_recursive_doubling(const void *sendbuf, int send
 int MPIR_Iallgather_sched_intra_ring(const void *sendbuf, int sendcount, MPI_Datatype sendtype,
                                      void *recvbuf, int recvcount, MPI_Datatype recvtype,
                                      MPIR_Comm * comm_ptr, MPIR_Sched_t s);
-int MPIR_Iallgather_intra_recexch_distance_doubling(const void *sendbuf, int sendcount,
+int MPIR_Iallgather_intra_gentran_recexch_doubling(const void *sendbuf, int sendcount,
```

Comment:
All of these should probably be prefixed with `tsp`, not `gentran`, right? Transport is the thing being used. Gentran is the specific transport (like `netmod` vs. `ofi`).

Reply:
These functions are entrant functions that is not under tsp namespace. They are there to be individually selected. When we add `othertran`, we will need another entrant function to select `othertran_recexch_...`.

Reply:
👍 Since I'm not as familiar with the collective code, that's why I asked @zhenggb72 to review this.